<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QWidget" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <widget class="QPushButton" name="pushButton_saveImage">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>save Image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ImageName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_iterations">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Iterations made</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
